src/modules/announcements/pages/Announcement.jsx

```diff
@@ -1,3 +1,67 @@
+import { useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import ReactStars from "react-rating-stars-component";
+import { ANNOUNCEMENTS_LIST } from "../../../constants/announcements";
+import { Button } from "../../../components/Button/Button";
+import styles from "../components/Announcement/Announcement.module.scss";
+import { DUMMY_LOGINS } from "../../../mock/auth";
+import SampleUser from "../../../assets/image/SampleUser/SampleUser.png";
+
 export const Announcement = () => {
-  return <div>announcement</div>;
+  const [phoneNumber, setPhoneNumber] = useState("XxX xXx XxX");
+  const announcementId = useParams().id;
+  const announcement = ANNOUNCEMENTS_LIST.find(
+    (announcementItem) => +announcementId === announcementItem.id,
+  );
+  const user = DUMMY_LOGINS.find(
+    (userItem) => userItem.id === announcement.userId,
+  );
+  return (
+    <div className={styles.centered__announcement}>
+      <section className={styles.announcement}>
+        <div className={styles.announcement__image}>
+          <img src={announcement.image} alt={announcement.title} />
+        </div>
+        <div className={styles.announcement__contactContainer}>
+          <div className={styles.announcement__contact}>
+            <p>{user?.name}</p>
+            <img src={user.image ? user.image : SampleUser} alt="" />
+            {user.company && <p>{user.company}</p>}
+            <Link to={`/announcements/user/${announcement.userId}`}>
```

Comment:
Move it before component to the const `ANNOUCEMENT_ROUTE` and make here string literal `${ANNOUCEMENT_ROUTE}${annoucement.userId}`